src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java

```diff
@@ -297,6 +297,23 @@ private void testAjaxPictureUrl() {
         return loginAdminToPage(profileUrl, typeOfPage);
     }
 
+    private <T extends AppPage> T loginAsInstructorToPage(String instructorId, AppUrl url, Class<T> typeOfPage) {
+        //logout
+        logout();
+
+        //login
+        getHomePage().clickInstructorLogin();
```

Comment:
Can we navigate to the `url` directly instead? This will avoid sending an additional request for the instructor home page.

Reply:
OK